docs/content/guide/migration.ngdoc

```diff
@@ -15,6 +15,29 @@ which drives many of these changes.
 * Several new features, especially animations, would not be possible without a few changes.
 * Finally, some outstanding bugs were best fixed by changing an existing API.
 
+## Migrating from 1.7 to 1.8
+
+Generally updating to 1.8.0 from 1.7.x should be a straightforward process and is highly recommended.
+AngularJS 1.8 is a breaking change release from 1.7 to mitigate a security issue.
+
+JqLite no longer turns XHTML-like strings like `<div /><span />` to nested elements
+`<div><span></span></div>`.
+Instead it will close the elements, leaving them as siblings: `<div></div><span></span>`.
```

Comment:
This sounds like the opposite of what happens:
- In 1.7, it would convert `<div /><span />` to `<div></div><span></span>`.
- In 1.8, it will leave them alone, which will result in the browser's converting them to `<div><span></span></div>`.

Reply:
Doh!!